airgun/entities/product.py

```diff
@@ -123,6 +123,27 @@ def advanced_sync(self, entities_list, sync_type):
         view.wait_for_result()
         return view.read()
 
+    def Verify_content_checksum(self, entities_list):
+        """Advanced Sync for product/products
+
+        :param entities_list: The product names to perform Advanced Sync action.
+        :param sync_type: value containing sync type.
+            eg: sync_type="optimized", sync_type="complete"
+        """
+
```

Comment:
You're missing the portion where you need to search for the product name and then selecting it.  Here's a sample code to get you started from `common.py` that is used in `contentview.py` in `view`.  Hope this helps!